annotation-web/src/main/java/eu/europeana/annotation/web/service/impl/BaseAnnotationValidator.java

```diff
@@ -571,7 +576,7 @@ private void validateSubtitleBody (Body body) throws PropertyValidationException
     }
     
     private void validateBodyExists(Body body) throws PropertyValidationException {
-    	if (body == null) {
+    	if (body == null  || body.getValue() == null) {
```

Comment:
the condition for linking can be added here:
if (webAnnotation.getBody() == null 
          || (webAnnotation.getBody().getHttpUri() == null && webAnnotation.getBody().getValue() == null))